### DISCUSSION TITLE
proxy host to IP/subfolder

### DISCUSSION THREAD

**Discussion by fpdragon** [NginxProxyManager/nginx-proxy-manager#3303] [Q&A] [answered]
Hi all,

maybe I am too stupid but I don't get it. I switched for apache reverse proxy configuration to nginx and GUI which seems much more modern. Also the certificate handling is much more convenient.

The problem is, I don't see how to configure a subpath for a proxy host on the Nginx GUI.

For example:

admin.mydomain.com
shall be rerouted to
http://192.168.1.10/admin

The "IP or Hostname" field and the "Port" field are both required. When I enter the data it is mixed up like:
http://192.168.1.10/admin:80
but I guess it should be:
http://192.168.1.10:80/admin

Maybe this is still possible with the "Advanced" tab but currently I am not sure how to configure this. 

Hope you can help.

Thanks.

**Comment by fpdragon** [comment, marked as answer]
I figured it out myself.

The trick is to define "Custom location" that overrules the root of the domain.

"Custom locations"
"Add"
"location" = "/"
"Forward Hostname / IP" = "192.168.1.10/admin"
"Port" = "80"

That's it.
As far as I understand, the "Hostname IP" on the "Details" tab is overruled by the "Custom location"

For me, it was kind of tricky to see the results. It is important to clear the browser cache, otherwise it can be hard to see if it is working...

**Comment by thk70** [comment]
Aren’t you being redirected to admin.mydomain.com/admin instead of forwarded to the subfolder?
/Tom
